src/InsetOrderOptimizer.cpp

```diff
@@ -251,6 +253,8 @@ void InsetOrderOptimizer::processHoleInsets()
             gcode_writer.setExtruder_addPrime(storage, gcode_layer, extruder_nr);
             gcode_layer.setIsInside(true); // going to print stuff inside print object
             gcode_layer.addPolygonsByOptimizer(hole_outer_wall, mesh_config.inset0_config, wall_overlapper_0, z_seam_config, wall_0_wipe_dist, spiralize, flow, retract_before_outer_wall);
+            // move inside so an immediately following retract doesn't occur on the outer wall
+            moveInside(wall_line_width_0 * 2);
```

Comment:
Ideally you'd use the line width in the `mesh_config.inset0_config` object, in case that is modified by the initial layer line width.